.travis.yml

```diff
@@ -1,31 +1,30 @@
 language: go
 sudo: false
-dist: trusty
-osx_image: xcode8.3
+dist: bionic
+osx_image: xcode10
 go:
-  - 1.10.x
   - 1.11.x
   - 1.12.x
+  - 1.13.x
 
 os:
   - linux
   - osx
 
+env:
+  GO111MODULE=on
```

Comment:
do we want to set `GOPROXY=https://proxy.golang.org` to speed things up?

Reply:
Good Idea! I must have missed it. I have added it now.